### PR TITLE
Update DislikeFill.js

### PR DESCRIPTION
**Pull request by britt6612**
Some reason when the new Dislike Fill for the correct direction was added the updated path was not pushed up when I added the SVG

### CODE REVIEW

File: src/js/__tests__/__snapshots__/Icon-test.js.snap [hunk 1]
```diff
@@ -7462,10 +7462,9 @@ exports[`Icon all icons 150`] = `
 >
   <path
     clip-rule="evenodd"
-    d="M11 24h-.997L10 23.003 11 23l-1 .003v-.261l-.002-.698-.006-2.207a2728.33 2728.33 0 0 1-.008-4.163c0-.42-.155-.83-.454-1.132C9.243 14.251 8.763 14 8 14H1.001L1 13v1H0V0h1v1-1h19c1.154 0 2.203.29 2.957 1.043C23.711 1.797 24 2.846 24 4v9.999L23 14h1v1h-7v5c0 1.154-.29 2.203-1.043 2.957C15.203 23.711 14.154 24 13 24h-2ZM5 12V2H2v10h3Z"
-    fill="none"
+    d="M13 24h.997l.003-.997L13 23l1 .003v-.261l.002-.698.006-2.207c.004-1.665.008-3.498.008-4.163 0-.42.155-.83.454-1.132.287-.291.767-.542 1.53-.542h6.999L23 13v1h1V0h-1v1-1H4C2.846 0 1.797.29 1.043 1.043.289 1.797 0 2.846 0 4v9.999L1 14H0v1h7v5c0 1.154.29 2.203 1.043 2.957C8.797 23.711 9.846 24 11 24h2Zm6-12V2h3v10h-3Z"
+    fill="#000"
```
Comment (taysea) [inline src/js/__tests__/__snapshots__/Icon-test.js.snap:7466]: Should this be fill="none" with stroke="#000"?

Comment (britt6612) [reply src/js/__tests__/__snapshots__/Icon-test.js.snap:7466]: let me double check still works

File: src/js/icons/DislikeFill.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ import { StyledIcon } from '../StyledIcon';
 const DislikeFill = forwardRef((props, ref) => (
   <StyledIcon ref={ref} viewBox="0 0 24 24" a11yTitle="Dislike" {...props}>
     <path
-      fill="none"
       stroke="#000"
+      fill='none'
```
Comment (taysea) [inline src/js/icons/DislikeFill.js:9]:
```suggestion
      fill="none"
```

**Review by taysea — APPROVED** [review]
Thanks for the fix.

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**